readme.txt

```diff
@@ -0,0 +1,36 @@
+=== Switch to ClassicPress from WordPress ===
+Contributors: classicpress
+Tags: classicpress, classic-press, classic, press
```

Comment:
Can we add gutenberg? (are we allowed to mention other plugin names?)

Reply:
It is considered Black Hat SEO and is not recommended initially.

Furthermore, only 5 tags are "really" allowed and published; the others are not visible, so it is better to insert only 5 of the most important ones initially.

https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/#12-public-facing-pages-on-wordpress-org-readmes-must-not-spam/